txpipe/twopoint.py

```diff
@@ -1429,6 +1429,164 @@ def write_output_sacc(self, data, meta, results):
 
         # Also make a plot of the data
 
+class TXGammaTRandoms(TXTwoPoint):
+    """
+    This subclass of the standard TXTwoPoint uses the centers
+    of stars as "lenses", as a systematics test.
+    """
+    name = "TXGammaTRandoms"
+    inputs = [
+        ('shear_catalog', MetacalCatalog),
+        ('shear_tomography_catalog', TomographyCatalog),
+        ('shear_photoz_stack', HDFFile),
+        ('lens_tomography_catalog', TomographyCatalog),
+        ('lens_photoz_stack', HDFFile),
+        ('random_cats', RandomsCatalog),
+        ('star_catalog', HDFFile),
+        ('patch_centers', TextFile),
+    ]
+    outputs = [
+        ('gammat_randoms', SACCFile),
+        ('gammat_randoms_plot', PNGFile),
+    ]
+    # Add values to the config file that are not previously defined
+    config_options = {
+        'calcs':[0,1,2],
+        'min_sep':2.5,
+        'max_sep':100,
+        'nbins':20,
+        'bin_slop':0.1,
+        'sep_units':'arcmin',
+        'flip_g2':True,
+        'cores_per_task':20,
+        'verbose':1,
+        'reduce_randoms_size':1.0,
```

Comment:
Are we still subtracting randoms?  Not sure if that makes sense in this context!  This will be in the superclass.